cpp/include/cudf/functions.h

```diff
@@ -949,3 +949,26 @@ gdf_error gdf_order_by(gdf_column** input_columns,
                        size_t       num_inputs,
                        gdf_column*  output_indices,
                        int          flag_nulls_are_smallest);
+
+/* --------------------------------------------------------------------------*
+ * @brief Finds the indices of the bins in which each value of the column
```

Comment:
This description is a bit too long for a `@brief`. Try to stick to a single sentence description, then put the more detailed description in a new paragraph after your `brief`.